bin/update-ips.bash

```diff
@@ -104,8 +105,9 @@ checkIfDiffAndUpdateKubectlIPs() {
   local config_key="${3}"
   local config_file="${4}"
 
-  local -a ips
-  readarray -t ips <<< "$(getKubectlIPs "${cluster}" "${label}" | tr ' ' '\n')"
+  local -a ips kubectlIPs
+  kubectlIPs=$(getKubectlIPs "${cluster}" "${label}")
+  readarray -t ips <<< "$(echo "${kubectlIPs}" | tr ' ' '\n')"
```

Comment:
TIL:

```console
$ readarray --help
...
    A synonym for `mapfile'.
```

I typically use `mapfile` so I was wondering what the difference bewteen these were